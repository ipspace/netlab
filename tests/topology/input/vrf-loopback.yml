# Regression test for #1108
defaults:
  device: eos

module: [ vrf ]

vrfs:
  red:
  blue:

vrf.loopback: True

nodes:
  r1:
    vrfs:
      red:
      blue:
