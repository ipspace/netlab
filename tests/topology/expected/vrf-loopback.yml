input:
- topology/input/vrf-loopback.yml
- package:topology-defaults.yml
module:
- vrf
name: input
nodes:
  r1:
    af:
      ipv4: true
    box: arista/veos
    device: eos
    id: 1
    interfaces:
    - ifindex: 10001
      ifname: Loopback1
      ipv4: 10.2.0.1/32
      name: VRF Loopback red
      neighbors: []
      type: loopback
      virtual_interface: true
      vrf: red
    - ifindex: 10002
      ifname: Loopback2
      ipv4: 10.2.0.2/32
      name: VRF Loopback blue
      neighbors: []
      type: loopback
      virtual_interface: true
      vrf: blue
    loopback:
      ifindex: 0
      ifname: Loopback0
      ipv4: 10.0.0.1/32
      neighbors: []
      type: loopback
      virtual_interface: true
    mgmt:
      ifname: Management1
      ipv4: 192.168.121.101
      mac: 08:4f:a9:00:00:01
    module:
    - vrf
    name: r1
    vrf:
      as: 65000
      loopback: true
    vrfs:
      blue:
        af:
          ipv4: true
        export:
        - '65000:2'
        id: 2
        import:
        - '65000:2'
        loopback_address:
          ipv4: 10.2.0.2/32
        networks:
        - ipv4: 10.2.0.2/32
        rd: '65000:2'
        vrfidx: 101
      red:
        af:
          ipv4: true
        export:
        - '65000:1'
        id: 1
        import:
        - '65000:1'
        loopback_address:
          ipv4: 10.2.0.1/32
        networks:
        - ipv4: 10.2.0.1/32
        rd: '65000:1'
        vrfidx: 100
provider: libvirt
vrf:
  as: 65000
  loopback: true
vrfs:
  blue:
    export:
    - '65000:2'
    id: 2
    import:
    - '65000:2'
    rd: '65000:2'
  red:
    export:
    - '65000:1'
    id: 1
    import:
    - '65000:1'
    rd: '65000:1'
